### PR TITLE
Recast README into MarkDown with various minor corrections and improvements

### PR DESCRIPTION
**Pull request by anthonyeleven**
Recast README into MarkDown with various minor spelling and formatting corrections.

Fixes: https://github.com/redhat-performance/tuned/issues/535

Signed-off-by: Anthony D'Atri [anthonyeleven@users.noreply.github.com](mailto:anthonyeleven@users.noreply.github.com)

### CODE REVIEW

**Comment by yarda** [comment]
Thanks. It's definitely improvement. There are still some outdated information, but we will update it later.

**Comment by yarda** [comment]
@anthonyeleven could you please also update the Makefile? Just renaming README to README.md (on multiple places) there should work, because now the build is broken.

**Comment by anthonyeleven** [comment]
@yarda I've pushed a Makefile change, PTAL

**Comment by yarda** [comment]
@anthonyeleven hmm I still cannot see the Makefile update in the PR and the CI build is still failing with "cp: cannot stat 'README': No such file or directory", e.g.:
https://download.copr.fedorainfracloud.org/results/packit/redhat-performance-tuned-534/fedora-38-x86_64/06035130-tuned/builder-live.log.gz

**Comment by anthonyeleven** [comment]
Grr.  https://programmerhumor.io/wp-content/uploads/2023/04/programmerhumor-io-linux-memes-backend-memes-499cf62c592fd27.jpg

(try now)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+version, please check out branch '1.0' in our Git repository.)
+
+How to use it
+-------------
+
+TuneD is incompatible with ``cpupower`` and ``power-profiles-daemon``. If you
+have these services, uninstall or disable them.
+
+On Fedora, Red Hat Enterprise Linux, and their derivatives: install the ``tuned`` package
+(optionally ``tuned-utils``, ``tuned-utils-systemtap``, and ``tuned-profiles-compat``)::
```
Comment (yarda) [inline README.md:13]: Double "::". It's on more places and in GitHub preview it renders as double "::". Is it intended?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+To recommend a profile for a given system, run::
+
+```bash
+  # tuned-adm recommend
+```
+Currently only static detection is
+implemented - it decides according to data in ``/etc/system-release-cpe`` and
+the output of ``virt-what``. The rules for autodetection are defined in the file
+``/usr/lib/tuned/recommend.d/50-tuned.conf``. They can be overridden by the user by
+creating a file in ``/etc/tuned/recommend.d`` or a file named ``recommend.conf` in
```
Comment (yarda) [inline README.md:97]: Missing "`" to close "recommend.conf".

**Review by yarda — CHANGES_REQUESTED** [review]
The CI passes now, thanks. It mostly LGTM, but I spotted two minor things:
- double colons "::". Is it intended? GitHub renders it as double colons and I think it looks weird, but I am not against it if there is some standard or convention to write it this way
- there is one missing closing tag causing that the text is misrendered, see the inline comment and please fix it

File: README.md (hunk 2 above)
Comment (anthonyeleven) [reply README.md]: oops, fixed

File: README.md (hunk 1 above)
Comment (anthonyeleven) [reply README.md]: Hmmm, that must be a Sphinx thing, that's the way we do it for Ceph docs.  I've reverted to single colons here.

**Comment by yarda** [comment]
LGTM, thanks.

**Pull request merged**